src/Breakpoint/index.d.ts

```diff
@@ -0,0 +1,2 @@
+export { breakpointObserver } from "./breakpointObserver";
+export { breakpoints } from "./constants";

```

Comment:
Missing the `Breakpoint` Svelte component.

```suggestion
export { default as Breakpoint } from "./Breakpoint.svelte";
export { breakpointObserver } from "./breakpointObserver";
export { breakpoints } from "./constants";

```